core/src/test/java/org/infinispan/persistence/file/SingleFileStoreCompatibilityTest.java

```diff
@@ -48,6 +52,23 @@ protected void beforeStartCache(Version version) throws Exception {
       Files.copy(is, sfsFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
    }
 
+   @Override
+   public void testReadWriteFrom11() throws Exception {
+      // Execute the tests as normal
+      super.testReadWriteFrom11();
+
+      // Then restart the CacheManager to ensure that the entries are still readable on restart
+      cacheManager.stop();
```

Comment:
I would change this test in the abstract class. 
In the abstract class, the test would run in all cache stores  we have :)

Reply:
Done. Let's see what CI thinks on https://github.com/infinispan/infinispan/pull/9409